main.js

```diff
@@ -0,0 +1,62 @@
+// the book class
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+// to create and add the book
+function creatAndAddBooks(book, ul) {
+  // create list li elemnts
+  const li = document.createElement('li');
+  li.className = `${book.title}${book.author}`;
+  const pForTitle = document.createElement('p');
+  const pForAuthor = document.createElement('p');
+  const pForDelete = document.createElement('button');
+  const hr = document.createElement('hr');
+  pForDelete.className = 'delete';
+  // add content to the elements
+  pForTitle.textContent = book.title;
+  pForAuthor.textContent = book.author;
+  pForDelete.textContent = 'Delete';
+  // add element to the list and the list to the ul
+  li.appendChild(pForTitle);
+  li.appendChild(pForAuthor);
+  li.appendChild(pForDelete);
+  li.appendChild(hr);
+  ul.appendChild(li);
+}
+// create the book list
+const books = [];
+// call nedded elements from the document
+const bookList = document.querySelector('.book-list');
+const form = document.getElementById('abbBookForm');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+// create the books, add them the book list and local storage
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const newBooke = new Book(title.value, author.value);
+  books.push(newBooke);
+  creatAndAddBooks(newBooke, bookList);
+  localStorage.books = JSON.stringify(books);
+});
```

Comment:
- [ ] you are adding the book to the local storage correctly which is great 👏 however when reloading the page all the books that were saved disappear anyway, this is due to the fact that you are not getting your data from the local storage when reloading the page. please make sure to fix this before requesting for another code review.